controllers/progressivesync_controller.go

```diff
@@ -173,6 +173,8 @@ func (r *ProgressiveSyncReconciler) requestsForApplicationChange(o client.Object
 				Namespace: pr.Namespace,
 				Name:      pr.Name,
 			}})
+			message := fmt.Sprintf("App name: %s; App status: %s", app.Name, &app.Status)
```

Comment:
The whole app status is quite big, maybe we only care about the sync status and health status?